elsasserlib/R/normGeneCounts.R

```diff
@@ -0,0 +1,48 @@
+
+#' Quantile Normalization of Gene Count Matrix
+#'
+#' This function replaces original values with estimated values according 
+#' to the rank within each sample.
+#'
+#' @param gene.counts Gene count table for normalization
+#' @return Normalised Counts
+#' @export
+quantileNorm <- function(gene.counts)
```

Comment:
`snake_case` styling is preferred for naming: `quantile_norm`. This is of course a style decision, but consistency is relevant.